pallets/admin-utils/src/lib.rs

```diff
@@ -104,6 +118,9 @@ pub mod pallet {
             Ok(())
         }
 
+        /// The extrinsic sets the serving rate limit for a subnet.
+        /// It is only callable by the root account.
```

Comment:
Please fix this line for each call that allows the subnet owner OR the root account